content/blogposts/2022/release-3.41.md

```diff
@@ -0,0 +1,58 @@
+- Trade-off a little more setup for the Sourcegraph admin, against less setup for users (no local `src-cli`)
+
+This is the outcome of months of iteration on the experimental version of the feature, and we're grateful to early users for their feedback!
+
+This feature requires to setup [executors](https://docs.sourcegraph.com/admin/executors) (much like CI agents), which Sourcegraph will use to offload expensive tasks. Executors can also be used to run Code Intelligence [auto-indexing](https://docs.sourcegraph.com/code_intelligence/how-to/enable_auto_indexing) (experimental).
+
+<!-- TODO: asset welcome right after deployment on demo -->
+
+###  Batch specs can now mount local files on steps containers (experimental)
+
+As we improve Batch Changes, we're heavily focusing on making it easy to write and debug batch specs. Iteration time needs to be as small as possible, so that users can quickly put together a spec, run it, see results, and quickly improve on it.
+
+Many users have custom tooling or scripts that they run as batch changes steps. The only way to do that until now was to bake those scripts into the step container, or to copy paste the scripts directly in the spec using [steps.files](https://docs.sourcegraph.com/batch_changes/references/batch_spec_yaml_reference#steps-files). Both options slow down iterating on batch changes, requiring either to rebuild the container at each modification, or to copy paste code inside YAML.
+
+Now you can directly mount files or directories on the batch change container using [steps.mount](https://docs.sourcegraph.com/batch_changes/references/batch_spec_yaml_reference#steps-mount). This is an experimental feature that's only supported when running batch changes locally. Lots of improvements are on the way ([issue](https://github.com/sourcegraph/sourcegraph/issues/14851)).
+
+We're committed to making iterating on batch changes as fast as possible. Feedback welcome on what slows you down!
+
+## Sort and filter "detect and track patterns" code insights series 
+
+<!-- TODO: add screenshot of sorting --> 
+
+Detect and track patterns charts now allow setting the series sort order and number (up to 20) of series. This means you can set an insight with greater than 20 match patterns to show the 1-20 most or least common series by match count; oldest or newest version by series name; or oldest-added or newest-added versions by date appearing in the codebase. Previously, if there were greater than 20 match patterns, it returned a random set of 20 each session.
+
+## Sourcegraph Extensions
+
+### Allow only extensions authored by Sourcegraph
+
+You can now restrict users to only use Sourcegraph-authored extensions by setting [`extensions.allowOnlySourcegraphAuthoredExtensions`](../config/site_config.md) to `true` in your site configuration. This allows you to easily enforce that any extensions you are using are built and supported by Sourcegraph.
```

Comment:
@ryankscott, what's the customer impact for this? Are non-native extensions seen as dangerous/untrustworthy?

Reply:
@iskyOS they _may_ be untrustworthy, they are open source so you can inspect / scan the code to determine. More importantly if you're using our extensions you'll get our support, be covered by our SLAs and contractual guarantees etc.

Reply:
@abschu @ryankscott  What do you think about adding an explicit mention of the SLA here? Something like: 
"While we encourage trying out open source extensions from the community, this setting may be valuable for those teams who want to ensure all of their extensions are covered by Sourcegraph support and SLAs."

Reply:
```suggestion
You can now restrict users to only use Sourcegraph-authored extensions by setting [`extensions.allowOnlySourcegraphAuthoredExtensions`](../config/site_config.md) to `true` in your site configuration. This allows you to easily enforce that any extensions you are using are built and supported by Sourcegraph. While we encourage trying out open source extensions from the community, this setting may be valuable for those teams who want to ensure all of their extensions are directly supported by Sourcegraph.
```